### PR TITLE
maximum to torch frontend

### PR DESCRIPTION
**Pull request by mazy06000**
#3898

### CODE REVIEW

**Comment by mazy06000** [comment]
Hello @RashulChutani, let me know if you think the code is correct.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py [hunk 1]
```diff
@@ -650,6 +650,47 @@ def test_torch_not_equal(
...
+@given(
+    dtype_and_inputs=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_numeric_dtypes).intersection(
+                set(ivy_torch.valid_numeric_dtypes)
+            ),
+        ),
+        num_arrays=2,
+        allow_inf=False,
+        shared_dtype=True,
```
Comment (CheezBallzPi) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py:664]: I think maximum supports differing dtypes, but check yourself to be sure.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py [hunk 2]
```diff
@@ -650,6 +650,47 @@ def test_torch_not_equal(
...
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_tree="maximum",
+        input=np.asarray(inputs[0], dtype=input_dtype[0]),
```
Comment (CheezBallzPi) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py:688]: The input should be a torch tensor, not a numpy array.

File: ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py [hunk 3]
```diff
@@ -650,6 +650,47 @@ def test_torch_not_equal(
...
+        shared_dtype=True,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.torch.maximum"
+    ),
+)
+def test_torch_maximum(
+    dtype_and_inputs,
+    as_variable,
+    with_out,
```
Comment (CheezBallzPi) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_comparison_ops.py:673]: `with_out` needs to be defined in `@given`.

**Review by CheezBallzPi — CHANGES_REQUESTED** [review]
Reviewed.

**Pull request closed without merging**